src/components/toolbar/mdc-toolbar.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <header class="mdc-toolbar">
+    <!-- here are some sugar slots -->
+    <div class="mdc-toolbar__row" v-if="$slots.icon || $slots.title" ref="firstRow">
+      <section class="mdc-toolbar__section mdc-toolbar__section--align-start" v-if="$slots.icon">
+        <slot name="icon" />
+      </section>
+      <section class="mdc-toolbar__section mdc-toolbar__section--align-start" v-if="$slots.title">
+         <slot name="title" />
+      </section>
+    </div>
+    <!-- if the user doesn't want to use the sugar slots then they can inject their own content here -->
+    <slot/>
+  </header>
+</template>
+
+<script lang="babel">
+  import MDCToolbarFoundation from '@material/toolbar/foundation'
+  import * as util from '@material/toolbar/util'
+
+  export default {
+    name: 'Toolbar',
+    props: ['titleElement', 'flexibleRowElement', 'fixedAdjustElement', 'firstRowElement'],
+    data () {
+      return {
+        classes: {},
+        foundation: null
+      }
+    },
+    methods: {
+      /**
+       * Init the foundation manually if it exists.
+       * Should be useful for testing/debugging.
+       *
+       * @return {void}
+       */
+      initFoundation () {
+        if (this.foundation) {
+          this.foundation.init()
```

Comment:
No if there is a problem with the adapter, in some cases it will be null. I ran into this case when testing.